bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -26,6 +26,9 @@ limitations under the License.
 
```

Comment:
this profile should be added to a new kokoro config corresponding to a dataflow presubmit check so the tests will run as part of this PR

Reply:
Sure, will do.